bin/asset-size-tracking/src/parse-modules.js

```diff
@@ -0,0 +1,42 @@
+const Library = require('./library');
+const moduleNames = ['ember-data', '@ember-data', '@ember/ordered-set', 'ember-inflector'];
+
+module.exports = function parseModules(builtAsset) {
```

Comment:
I think using the concat-stats output would make this massively easier. See a sample script that I wrote recently that does _similar_ things:

https://gist.github.com/rwjblue/fd5936b924748054cb65365da918e5c7

Reply:
Going to close this but definitely open to a refactor to use concat stats at some point if we gain additional capabilities from doing so. Currently we don't.